recipes/ruff/build.sh

```diff
@@ -0,0 +1,25 @@
+set -ex
+
+# Set conda CC as custom CC in Rust
+export CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER=$CC
+
+# Print Rust version
+rustc --version
+
+# Install cargo-bundle-licenses
+export CARGO_HOME="$BUILD_PREFIX/cargo"
+mkdir $CARGO_HOME
+cargo install cargo-bundle-licenses
```

Comment:
Please add `cargo-bundle-licenses` to `build` instead of building it from source.